packages/pwashortcut.rb

```diff
@@ -0,0 +1,185 @@
+require 'package'
+
+class Pwashortcut < Package
+  description 'Scripts to create a PWA shortcut on ChromeOS dock for Chromebrew GUI Apps.'
+  homepage 'https://github.com/supechicken666/pwashortcut/'
+  version '1.1'
+  source_url 'https://github.com/supechicken666/pwashortcut/archive/1.0-stable.tar.gz'
+  source_sha256 '9e028cb35ce6b0ba840b476d4641be496686d4ee16c73935c48d1fb30ca70d07'
+
+  def self.build
+    system "pip install flask"
+    system "cat <<'EOF'> main.py.bak
+from flask import Flask, render_template, redirect, url_for, send_from_directory
+import os
+
+app = Flask(__name__)
+
+EOF"
+    #######################################
+    system "cat <<'EOF'> installer.html.bak
+<!DOCTYPE html>
+<html>
+<head>
+<title>Shortcut Installer</title>
+<link rel=\"manifest\" href=\"manifest.json\" />
+</head>
+<body style=\"background-color:powderblue;\">
+    <h1>Shortcut Installer</h1>
+    <p>Add Linux GUI App shortcut to your dock</p>
+    <h2>HowTo</h2>
+    <h3>Step 1</h3>
+    <p>Click the ••• icon in the right corner of the screen</p>
+    <h3>Step 2</h3>
+    <p>Select 'Create shortcut...' under 'More tools'</p>
+    <h3>Step 3</h3>
+    <p>Tick the 'Open as window' checkbox</p>
+    <h3>Step 4</h3>
+    <p>Simply click the blue 'Create' button!</p>
+    <h3>Step 5</h3>
+    <p>Close this window and reopen it by the new shortcut</p>
+    <h2>That's all:) </h2>
+</body>
+</html>
+EOF"
+    #######################################
+    system "cat <<'EOF'> manifest.json.bak
+v1
+  \"name\"v4 \"linuxapp\"v6
+  \"short_name\"v4 \"linuxapp\"v6
+  \"display\"v4 \"standalone\"v6
+  \"start_url\"v4 \"unixapp.app\"v6
+  \"theme_color\"v4 \"#000000\"v6
+  \"background_color\"v4 \"#000000\"v6
+  \"icons\"v4 v2
+    v1
+      \"src\"v4 \"icon/brew.png\"v6
+      \"sizes\"v4 \"546x546\"v6
+      \"type\"v4 \"image/png\"
+    v9
+  v8
+v9
+EOF"
+    system "sed -i 's/v6/,/g' manifest.json.bak"
+    system "sed -i 's/v4/:/g' manifest.json.bak"
+    system "sed -i 's/v1/\{/g' manifest.json.bak"
+    system "sed -i 's/v2/\[/g' manifest.json.bak"
+    system "sed -i 's/v9/\}/g' manifest.json.bak"
+    system "sed -i 's/v8/]/g' manifest.json.bak"
```

Comment:
I guess I don't understand why you need to run the sed commands.  Why not just create the manifest file verbatim?

Reply:
Because the `:` in manifest will break the quotation marks in ruby